packages/dds/tree/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -691,17 +694,19 @@ export class ModularChangeFamily
 			0x89a /* Unexpected destroys in change to invert */,
 		);
 
+		const revInfos: RevisionInfo[] = [{ revision: revisionForInvert }];
```

Comment:
Don't forget to populate the `rollbackOf` field with `change.revision` when `isRollback` is true.